components/tools/OmeroPy/src/omero/plugins/duplicate.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+#
+# Copyright (C) 2016 University of Dundee & Open Microscopy Environment.
+# All rights reserved.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+"""
+   Plugin for duplicating object graphs
+
+"""
+
+import sys
+
+from omero.cli import CLI, GraphControl
+
+HELP = """Duplicate OMERO data.
+
+Duplicate entire graphs of data based on the ID of the top-node.
+
+Examples:
+
+    # Duplicate a dataset
+    omero duplicate Datset:50
```

Comment:
At least that is in my code and not @pwalczysko's! I noiced it when copying it to the PR description but thought I could sneak it through/leave it in case there are other changes.